test/safe_math/safe_math.edl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+
+  struct CountSizeParamStruct {
+    uint64_t count;
+    size_t size;
+    [count=count, size=size] uint64_t* ptr;
```

Comment:
For deep-copyable structs, we allow both `count` and `size` to be specified. (It is an inconsistency that we support this while not supporting specifying both for ptr function parameters).
Do we ensure that size is a multiple of sizeof(Base type) of the pointer?

Reply:
No, we currently don't validate it; i.e., we don't restrict the use of the `size`.